react/react1/week3/todo-week3/src/registration/RegistrationItem.js

```diff
@@ -0,0 +1,35 @@
+import "./Registration.css";
+
+export const RegistrationItem = ({
+  name,
+  startDate,
+  endDate,
+  startTime,
+  endTime,
+}) => {
+  const endInMinutes = endTime.split(":");
```

Comment:
Perhaps this could be in a function that is named after  what it does

Reply:
yes 👍